docs/user/example-cases/dam-break/frontier.batch.base

```diff
@@ -23,10 +23,6 @@ source ${RDYCORE_DIR}/config/set_petsc_settings.sh --mach ${MACH} --config 1
 # in the <RDYCORE_DIR>/build-${PETSC_ARCH}.
 export RDYCORE_EXE=${RDYCORE_DIR}/build-${PETSC_ARCH}/bin/rdycore
 
-# Create a directory for the slurm job in which all log files would be saved
-export DEST_DIR=cpu.N_${N}.${SLURM_JOB_ID}
-mkdir ${DEST_DIR}
-
 # Set LD_LIBRARY_PATH (not sure if this is needed anymore)
```

Comment:
Can you remove this and see if things still work? It shouldn't be necessary as far as I know, because we write these paths into the executable (the RPATH method)

Reply:
Same error as in the case of Perlmutter

```
/lustre/orion/cli192/proj-shared/gb9/rdycore/rdycore/build-frontier-gpu-hdf5_1_14_3-opt-64bit-gcc-11-2-0-fc288817/bin/rdycore: error while loading shared libraries: libmuparser.so.2: cannot open shared object file: No such file or directory
```